Tools/Tasks-Repository/readme.md

```diff
@@ -1,7 +1,8 @@
 # Tasks-Repository
 author: Benji Kovacevic
 
-This solution contains Tasks Repository Watchlist and Playbook that are used to assign tasks automaticlly based on incident title. 
+This solution contains Tasks Repository Watchlist and Playbook that are used to assign tasks automaticlly based on incident title. <br>
+This solution is explained in details in this blog - <a href="https://techcommunity.microsoft.com/t5/microsoft-sentinel-blog/create-tasks-repository-in-microsoft-sentinel/ba-p/4038563">Create Tasks Repository in Microsoft Sentinel</a>.
```

Comment:
link redirects to article not found page.
<img width="917" alt="image" src="https://github.com/Azure/Azure-Sentinel/assets/104008048/59a26475-5724-4724-aa55-2faf3e576172">